website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,503 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.1 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are happy to announce the release of Metals v0.9.1, which concentrated on
+fixing issues with recently released Scala 3 support, Ammonite support and a
+couple of new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">261</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">96</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">16</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">3</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/24?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Ammonite support
+- new project provider
+- worksheet dependency support
+- Scala 2.13.3, 0.24.0, 0.25.0-RC1 support
+
+## Ammonite support
+
+Thanks to a great work by [alexarchambault](https://github.com/alexarchambault)
+we know support [Ammonite](https://ammonite.io) script files. Ammonite enables
+to create scripts that you can run easily from the command line, without the
+overhead of setting build tool while still allowing you to download and use
+dependencies.
+
+The support includes most of the LSP features such as diagnostics, completions,
+references, go to definition, but does not include running or debugging the
+scripts. This new feature works by starting an additional Ammonite BSP server
+that compiles the scripts and provides additional information about them such as
+scala version or information about the added dependencies. You can change the
+default JVM parameters for the server such as `-Xmx`, which will limit the used
+memory in Metals settings.
```

Comment:
```suggestion
default JVM parameters for the server such as `-Xmx`, which will limit the used
memory. These can be added via your metals editor extension using the newly created `metals.ammoniteJvmProperties` setting.
```

Reply:
I'm not sure on the wording here, but I thought it might be nice to be explicit about the setting for this.

Sort of related, what are you thoughts about mentioning the `metals.ammonite-start` and `metals.ammonite-stop` commands. The only reason I mention it is that we often add commands like this without mentioning them, so users may just be totally unaware of what they do if they see them. If we mention them at least they are aware they are there and they know what they do. I don't feel strongly about this, but just an idea.

Reply:
Good idea! I will add that!